bindings/python/pmix.pyx

```diff
@@ -251,6 +251,63 @@ cdef class PMIxClient:
             pmix_free_info(info, ninfo)
         return rc
 
+    # retrieve a value from the keystore
+    #
+    # @proc [INPUT]
+    #       - namespace and rank of the client (tuple)
+    #
+    # @key [INPUT]
+    #      - the key to be retrieved
+    #
+    # @dicts [INPUT]
+    #          - a list of dictionaries, where each
+    #            dictionary has a key, value, and val_type
+    #            defined as such:
+    #            [{key:y, value:val, val_type:ty}, … ]
+    def get(self, proc:tuple, ky, dicts:list):
+        cdef pmix_info_t *info;
+        cdef size_t ninfo;
+        cdef pmix_key_t key;
+        cdef pmix_value_t value;
+        cdef pmix_value_t *val_ptr;
+
+        ninfo   = 0
+        val_ptr = &value
```

Comment:
I'm sorry for missing some of these things. I promise this will be the last 🤷‍♂ 

PMIx_Get malloc's memory to return a pointer to the result. Thus, you don't need `value` in this function as that storage location will be ignored. Just set `val_ptr = NULL` here

Reply:
@rhc its ok, I figured the first one would take some back and forth. Just submitted an update.